notebooks/sam_tests/cmip6_density_transform_pipeline_dev.ipynb

```diff
@@ -8,7 +8,7 @@
     {
```

Comment:
<p>Yes it does. But can we integrate this in the <code>filter_incomplete_members</code>  logic? That way we have a single point of control that should check for all things a member should have or not have.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/cmip6_omz/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>